igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/service/TravisResultConverter.groovy

```diff
@@ -25,6 +25,9 @@ import groovy.util.logging.Slf4j
 class TravisResultConverter {
     static Result getResultFromTravisState(String state) {
         switch (state) {
+            case "created":
+                return Result.NOT_BUILT
+                break
             case "started":
                 return Result.BUILDING
                 break
```

Comment:
I've changed this so it returns IllegalArgumentException if the state is not handled by the different cases.